CHANGELOG.md

```diff
@@ -1,9 +1,34 @@
+## 2.8.0-beta.1
+
+### Features ✨ and improvements 🏁
+* Improve memory usage by reducing repeated line labels on overscaled tiles. ([#11414](https://github.com/mapbox/mapbox-gl-js/pull/11414))
+* Improve memory footprint by freeing memory more eagerly after loading tiles. ([#11434](https://github.com/mapbox/mapbox-gl-js/pull/11434))
+* Fix performance issue when placing many symbols on terrain. ([#11466](https://github.com/mapbox/mapbox-gl-js/pull/11466))
```

Comment:
```suggestion
* Improve performance when placing many symbols on terrain. ([#11466](https://github.com/mapbox/mapbox-gl-js/pull/11466))
```
So that it doesn't get misinterpreted as a bug fix/regression, as it's purely improvement.